### PR TITLE
Add correct paths for cri's systemd config files in CentOS.

### LINKED ISSUE #1378: 10-kubeadm.conf path is wrong for CentOS in cri-containerd.yaml playbook
The cri-containerd.yaml playbook searches for and replaces the KUBELET_EXTRA_ARGS variable in the wrong path for CentOS. It assumes the path is the same as Ubuntu(i.e., /etc/systemd/system/kubelet.service.d/10-kubeadm.conf), but as seen below, this is different in CentOS.

```
$ cat /etc/centos-release
CentOS Linux release 7.7.1908 (Core)
$ rpm -ql kubeadm
/usr/bin/kubeadm
/usr/lib/systemd/system/kubelet.service.d/10-kubeadm.conf
```

### PR DESCRIPTION
**Pull request by georgegoh**
Fixes #1378

### CODE REVIEW

File: contrib/ansible/cri-containerd.yaml [hunk 1]
```diff
@@ -31,17 +31,30 @@
...
 
-    - name: "Add runtime args in kubelet conf"
+    - name: "Add runtime args in kubelet conf (Ubuntu)"
       lineinfile:
         dest: "/etc/systemd/system/kubelet.service.d/10-kubeadm.conf"
         line: "Environment=\"KUBELET_EXTRA_ARGS= --runtime-cgroups=/system.slice/containerd.service --container-runtime=remote --runtime-request-timeout=15m --container-runtime-endpoint=unix:///run/containerd/containerd.sock\""
         insertafter: '\[Service\]'
-      when: check_args.stdout == ""
-    
+      when: andisble_distribution == "Ubuntu" and check_args.stdout == ""
```
Comment (AkihiroSuda) [inline contrib/ansible/cri-containerd.yaml]: typo

**Comment by AkihiroSuda** [comment]
Please sign the commit (`git commit -a -s --amend`)

**Comment by georgegoh** [comment]
Signed and typos fixed.

**Comment by mikebrow** [comment]
looks like you might have had a problem rebasing your pr branch to your master after merging git commits changes to your master?

**Comment by georgegoh** [comment]
Yes. My git-fu is not great. I'll try again.

**Comment by georgegoh** [comment]
Closing this and created a clean new PR #1382

**Pull request closed without merging**